crates/remote/src/transport/ssh.rs

```diff
@@ -701,7 +705,15 @@ impl SshRemoteConnection {
                     .run_command(
                         self.ssh_shell_kind,
                         "wget",
-                        &[url, "-O", &tmp_path_gz.display(self.path_style())],
+                        &[
+                            "--connect-timeout",
+                            CONNECT_TIMEOUT_SECS,
+                            "--tries",
+                            "1",
```

Comment:
By default, `wget` retries up to 20 times! Given we don't retry at all with `curl` (which we always use over `wget` when available), I think we can disable this here. I'd rather fall back to the client-side download as soon as possible.